src/main/java/com/exasol/projectkeeper/validators/pom/PomFileValidator.java

```diff
@@ -25,8 +20,8 @@
             new SurefirePluginPomValidator(), new JacocoPluginPomValidator(), new FailsafePluginPomValidator(),
             new GpgPluginValidator(), new DeployPluginValidator(), new NexusStagingPluginValidator(),
             new JacocoAgentDependencyValidator(), new DependencyPluginValidator(), new SourcePluginValidator(),
-            new JavadocPluginValidator(), new ErrorCodeCrawlerPluginValidator(),
-            new ReproducibleBuildPluginValidator());
+            new JavadocPluginValidator(), new ErrorCodeCrawlerPluginValidator(), new ReproducibleBuildPluginValidator(),
+            new JarPluginExclusionValidator());
```

Comment:
Can we remove Exclusion from the class name? It's just an implementation detail, I think